nodes/widgets/ui_switch.js

```diff
@@ -4,6 +4,7 @@ const { appendTopic } = require('../utils/index.js')
 module.exports = function (RED) {
     function SwitchNode (config) {
         // create node in Node-RED
+        config.passthru = false
```

Comment:
This is in the `ui_switch.js` - think this should be in `ui_form.js`, no?

Reply:
Yes, it is in the wrong place, sorry...